src/components/modal/v2/parts/views/LongTerm/Content.jsx

```diff
@@ -147,21 +148,27 @@ export const LongTerm = ({
     return (
         <Fragment>
             <style>{styles._getCss()}</style>
-            <div className="content__row dynamic">
+            <div className={`content__row dynamic ${useNewCheckoutDesign ? 'checkout' : ''}`}>
                 <div className="content__col">
                     <Calculator
                         setExpandedState={setExpandedState}
                         calculator={calculator}
                         aprDisclaimer={offerAPRDisclaimers}
                         useV4Design={useV4Design}
+                        useNewCheckoutDesign={useNewCheckoutDesign}
                     />
                     <div className={`content__col ${expandedState ? '' : 'collapsed'}`}>
                         <div className="branded-image">
                             {/* TODO: include Icon component when desktop images are final */}
                         </div>
                     </div>
                 </div>
-                <Instructions instructions={instructions} useV4Design={useV4Design} expandedState={expandedState} />
+                <Instructions
+                    instructions={instructions}
+                    useV4Design={useV4Design}
+                    useNewCheckoutDesign={useNewCheckoutDesign}
+                    expandedState={expandedState}
+                />
             </div>
             <div className={`content__row disclosure ${expandedState ? '' : 'collapsed'}`}>
```

Comment:
```suggestion
                <Instructions
                    instructions={instructions}
                    useV4Design={useV4Design}
                    useNewCheckoutDesign={useNewCheckoutDesign}
                    expandedState={expandedState}
                />
            </div>
                           <div
                className={`content__row disclosure ${expandedState ? '' : 'collapsed'}  ${
                    useNewCheckoutDesign ? 'checkout' : ''
                }`}
            >
```
Let's add a checkout class conditional on the `content__row disclosure` div so we can update some of the spacing.

Reply:
@surekhaw Just wanted to call out one more thing for the Pay Monthly sticky button. Let's make sure to add the styles to the `LongTerm/styles.scss` file as well similar to what exists in the short term version:
```
.button__fixed-wrapper {
    @include fixedWrapper;
    bottom: 0;
    z-index: 1;

    @include tablet {
        border-radius: 0 0 $content-wrapper-border-radius $content-wrapper-border-radius;
        overflow: hidden;

        @include lander {
            border-radius: 0;
        }
    }
}

.button__container {
    display: flex;
    flex-direction: column;
    align-items: center;

    @include tablet {
        margin-bottom: 0;
        max-width: $content-wrapper-max-width-tablet;
        padding: 15px 0;
        .button {
            width: calc(80%);
        }
    }

    @include mobile {
        margin-bottom: 0;
        max-width: $content-wrapper-max-width-tablet;
        padding: 15px 0;
        .button {
            width: calc(80%);
        }
    }

    @include apiIframe {
        max-width: $content-wrapper-max-width-api-iframe;
    }
}

.checkout {
    background: $ql-gray;
}
```

Reply:
I just realized I put my above comment on the wrong comment thread, sorry for the confusion.